examples/orderbook/create-listing-with-nextjs/playwright.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig, devices } from "@playwright/test";
+
+export default defineConfig({
+    testDir: "./tests",
```

Comment:
**Suggestion:** The majority of the SDK project and also the examples use 2 space indentation. Could we follow the same for this example project?

Reply:
@lfportal Fixed.